frontend/src/wizards/Policy/PolicyWizard.tsx

```diff
@@ -177,7 +176,7 @@ export function PolicyWizard(props: {
           existingClusterSetBindings={props.clusterSetBindings}
           bindingSubjectKind={PolicyKind}
           bindingSubjectApiGroup={PolicyApiGroup}
-          defaultPlacementKind={props.defaultPlacementKind ?? PlacementRuleKind}
+          defaultPlacementKind={PlacementKind}
```

Comment:
Optional/Future: It looks to me like GRC is the only set of wizards that uses the `PlacementSection` component, so we can get rid of this property entirely here and make changes there.